bridge/opencensus/metric.go

```diff
@@ -38,26 +43,57 @@ type exporter struct {
 }
 
 // NewMetricExporter returns an OpenCensus exporter that exports to an
-// OpenTelemetry exporter.
+// OpenTelemetry (push) exporter.
 func NewMetricExporter(base metric.Exporter, res *resource.Resource) metricexport.Exporter {
-	return &exporter{base: base}
+	return &exporter{base: base, res: res}
 }
 
 // ExportMetrics implements the OpenCensus metric Exporter interface by sending
 // to an OpenTelemetry exporter.
 func (e *exporter) ExportMetrics(ctx context.Context, ocmetrics []*ocmetricdata.Metric) error {
 	otelmetrics, err := internal.ConvertMetrics(ocmetrics)
 	if err != nil {
-		return err
+		otel.Handle(err)
```

Comment:
This was a bug I found while adding unit tests.  This allows the bridge to have partial success sending metrics, and keeps the behavior of the previous OC bridge implementation.